src/object_functions.c

```diff
@@ -0,0 +1,2016 @@
+/* The comment below is needed for this file to be picked up by generate_ld */
```

Comment:
It looks like the differ thinks you made a new file and deleted unknown_0348C0.c rather than renaming it. Can you please fix this so I can see the changes?

Reply:
Hmm... I don't know how to fix this. Both git & github say that the file was renamed in the `Renamed unknown_0348C0 to object_functions` commit, but doesn't in the files section. This might not be a perfect solution, but you can view all the changes here: https://www.diffchecker.com/9D9csbFA. Do note that I'm still working on this file, as I need to rename all the parameters in some of the init/loop functions.

Reply:
Ah bummer, alright thanks.